docs/cloud/satellites/gha-runners.md

```diff
@@ -17,22 +17,38 @@ These runners come with the Earthly CLI preinstalled and configured to use the s
 
 Satellite-based GitHub Actions runners can be enabled for a particular repository or for all repositories of a GitHub organization at once.
 
-The integration process requires you to provide us with a GitHub token, so we can:
+The integration process requires you to be an organization/repository admin, and to provide us with a GitHub token, so we can:
 - register a webhook in your GitHub repository/organization to receive events associated to GitHub Actions jobs
 - create GitHub self-hosted runners on demand, to process your repository/organization jobs
 
 Follow the next steps to create such integrations:
 
 ### 1. Create a GitHub token
-Both GitHub classic and fine-grained tokens are supported, but depending on the type of installation (organization-wide or single-repository), the provided token requires different scopes:
 
-| Integration type | User type          | Classic token scopes          | Fine-grained token permissions                                       | 
-|------------------|--------------------|-------------------------------|----------------------------------------------------------------------|
-| Organization     | Organization admin | `admin:org_hook`, `admin:org` | `organization_hooks:write`, `organization_self_hosted_runners:write` |
-| Repository       | Repository admin   | `admin:repo_hook`, `repo`     | `repository_hooks:write`, `administration:write`                     |
+- Go to https://github.com/settings/tokens/new to create a new GitHub classic token.
+
+- Give a name to your token that clearly shows its purpose, for example:
+
+  ![token name](./gha/token-name.png)
+
+- Set the token as non-expiring (notice that the integration won't work after the token expires)
+
+  ![token expiration](./gha/token-expiration.png)
+
+- Check the following scopes:
+  - For organization integrations: `admin:org`,`admin:org_hook`
+  - Alternatively, for repository integrations: `repo`,`admin:repo_hook`
+  
+- Click "generate token"
+
+  ![token generate](./gha/token-generate.png)
+
+- Copy the token value to use it in the following step
+
+  ![token copy](./gha/token-copy.png)
 
 {% hint style='info' %}
-Follow the [official docs](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens) for detailed information on how to create a GitHub token, and make sure to set an expiration long enough, since the integration won't work after the token expires.
+Alternatively, if you prefer creating a fine-grained tokens, make sure to set the following permissions for it: org: `organization_hooks:write`, `organization_self_hosted_runners:write`, repo: `repository_hooks:write`, `administration:write`
```

Comment:
Should we re-iterate here that this token will expire and they will need to come back and update it if they choose to use this method?